pep-0654.rst

```diff
@@ -767,40 +765,41 @@ it into the new ``ExceptionGroup``.
 
 .. code-block::
 
-   >>> try:
-   ...     try:
-   ...         raise ExceptionGroup("one", [ValueError('a'), TypeError('b')])
-   ...     except *ValueError:
-   ...         raise ExceptionGroup("two", [KeyError('x'), KeyError('y')])
-   ... except BaseException as e:
-   ...     traceback.print_exception(e)
-   ...
-   Traceback (most recent call last):
-     File "<stdin>", line 3, in <module>
-   ExceptionGroup
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 3, in <module>
-      ExceptionGroup: one
-        ------------------------------------------------------------
-        ValueError: a
-
-      During handling of the above exception, another exception occurred:
-
-      Traceback (most recent call last):
-       File "<stdin>", line 5, in <module>
-      ExceptionGroup: two
-        ------------------------------------------------------------
-        KeyError: 'x'
-        ------------------------------------------------------------
-        KeyError: 'y'
-
-      ------------------------------------------------------------
-      Traceback (most recent call last):
-       File "<stdin>", line 3, in <module>
-      ExceptionGroup: one
-        ------------------------------------------------------------
-        TypeError: b
+    >>> try:
+    ...     raise ExceptionGroup("one", [ValueError('a'), TypeError('b')])
+    ... except *ValueError:
+    ...     raise ExceptionGroup("two", [KeyError('x'), KeyError('y')])
+    ...
```

Comment:
The chaining display is not quite right. Note also that the counting is duplicated (1.1 appears twice).